src/python/grpcio/README.rst

```diff
@@ -107,9 +125,14 @@ Help, I ...
                     ^
     compilation terminated.
 
-  You can fix it by installing `python-dev` package. i.e
+  You can fix it by installing :code:`python-dev` package. i.e
 
   ::
 
     sudo apt-get install python-dev
+    
+  On macOS: append the path to the headers to :code:`CPPFLAGS`. For example:
+
+  ::
 
+    export CPPFLAGS="$CPPFLAGS -I/path/to/python/include/python3.10"
```

Comment:
TIL there's a pyenv-friendly way to get the current interpreter's include path: `python-config --include`. It can be used inline e.g. `export CPPFLAGS="$CPPFLAGS $(python-config --include)"`